invenio_records_resources/services/record.py

```diff
@@ -118,8 +132,10 @@ def read(self, id_, identity):
         """Retrieve a record."""
         pid, record = self.resolve(id_)
         self.require_permission(identity, "read", record=record)
-        links = ResourceUnitLinks(self.item_route, pid.pid_value).links()
-        # Todo: how do we deal with tombstone pages
+        links = self.linker.links(
+            "record", identity, pid_value=pid.pid_value, record=record
+        )
+        # TODO: how do we deal with tombstone pages
```

Comment:
Hasn't this been dealt with by @zzacharo (old comment that should be removed?)

Reply:
Oh git is weird: I just capitalized the existing `Todo` to `TODO` and it makes it show up like a new line... I haven't touched this aspect otherwise, so I would leave it until we finalize tombstones (#9 / #39)